### ISSUE TITLE
Impossible download multiple table in XLSX, also without virtual DOM

### ISSUE THREAD

**Issue by andrewittorio** [olifolkerd/tabulator#2965]
**Describe the bug**
This bug report is related to the report https://github.com/olifolkerd/tabulator/issues/2932#issue-678505951 that has already been closed. 
It seems that when we add a tabulator control as nested table it is not possible download its data in excel, also if the virtual DOM is disabled. 
As disabling the virtual Dom and doing different tests I always had the same error, I tried to debug tabulator.js to understand a bit deeply my problem. I noticed that I got an error at line 13839, because the object this.table is undefined.
As I saw that the object *modules* is available as class variable, in the file tabulator.js, I changed the line:

```
//>>BEGIN CHANGE
//this.table is undefined
//this.table.modules.comms.send(options.sheets[sheet], "download", "intercept", {
this.modules.comms.send(options.sheets[sheet], "download", "intercept", {
//END CHANGE<<
```

with this change my updated code (available at https://codepen.io/andrewittorio/full/qBZXEJd) works fine, but I am not sure this is a good solution.

**Tabulator Info**
4.7.2

**Working Example**
https://codepen.io/andrewittorio/pen/qBZXEJd

**To Reproduce**
Try to download the xlsx file with the button "Download all data". --> An exception is thrown: 
*tabulator.js:13839 Uncaught TypeError: Cannot read property 'modules' of undefined
    at Tabulator.xlsx (tabulator.js:13839)
    at Download.download (tabulator.js:13486)
    at Tabulator.download (tabulator.js:10616)
    at HTMLButtonElement.<anonymous> (pen.js:320)*.
 
If you try to change the line 13839 in tabulator.js:
//this.table.modules.comms.send(options.sheets[sheet], "download", "intercept", {
with the line:
this.modules.comms.send(options.sheets[sheet], "download", "intercept", {

it will be possible to download all the files in the example.

**Expected behavior**
Having multiple tabulator objects in my page, I would be able to use all of their data to create an excel file.

**Desktop (please complete the following information):**
 - OS: windows 10
 - Browser: chrome 
 - Version: 84

Thank you

**Comment by olifolkerd** [comment]
Hey @andrewittorio 

Thanks for the detailed bug report there, that was very helpful.

There was indeed a scoping issue there. in the last release the scope of the download functions was changed to the table and that one must have slipped through.

I have pushed an update to fix this to the 4.8 branch which will be released this coming weekend.

Cheers

Oli :)

**Comment by andrewittorio** [comment]
It's me that I thank you for this powerful tool!
